lib/elixir/src/elixir_bitstring.erl

```diff
@@ -351,4 +360,7 @@ format_error({bad_unit_argument, Unit}) ->
                 ['Elixir.Macro':to_string(Unit)]);
 format_error({bad_size_argument, Size}) ->
   io_lib:format("size in bitstring expects an integer or a variable as argument, got: ~ts",
-                ['Elixir.Macro':to_string(Size)]).
+                ['Elixir.Macro':to_string(Size)]);
+format_error({nested_match, Expr}) ->
+  io_lib:format("nested matching in bitstring is not supported, got: ~ts",
```

Comment:
I think "matching nested bitstrings is not supported" would be better because nested matching is possible in bitstrings: `<<a = b>> = bitstring`.

Reply:
@ericmj in fact, `<<a = b>> = "v"` is no allowed also.

Reply:
I would remove the word "nesting" and use "in match". Suggestion: "cannot pattern match inside a bitstring already in a match"